todo_app/data/trello_items.py

```diff
@@ -17,83 +17,105 @@ def get_items():
+        trello_default_get_header   = {"Accept": "*/*"}
+        trello_board_list_id_todo = os.environ.get('TRELLO_BOARD_LIST_ID_TODO','')
+        trello_board_list_id_done = os.environ.get('TRELLO_BOARD_LIST_ID_DONE','')
+        
+        get_item_req_url = trello_boards_url+trello_board_id+'/cards/?'+trello_api_security 
+        
+        try:
+            response = requests.get(get_item_req_url, data='',  headers=trello_default_get_header, verify=False)
+        except:
+            response = requests.get(get_item_req_url, data='',  headers=trello_default_get_header)
+
+        response_json = response.json()
+        print(response_json)
+        all_items = []
+
+        for item in response_json:
+            if item.get('idList','') == trello_board_list_id_todo:
+                all_items.append(Item(
+                        item.get('id',''),
+                        item.get('name',''),
+                        'todo'
+                    ))
+
+            elif item.get('idList','') == trello_board_list_id_done:
+                all_items.append(Item(
+                        item.get('id',''),
+                        item.get('name',''),
+                        'done'
+                    ))
+
+        return_items = all_items
+        session.clear()
+        return return_items
+    except Exception as e:
+        print("An error occurred during get_items:", e)
+        return []
 
-def add_item(title):
 
-    trello_cards_url        = "https://api.trello.com/1/cards"
-    trello_board_list_id_todo = os.environ.get('TRELLO_BOARD_LIST_ID_TODO','')
-    trello_api_key          = os.environ.get('TRELLO_API_KEY','')
-    trello_api_token        = os.environ.get('TRELLO_API_TOKEN','')
-    trello_default_post_header   = {"Accept": "application/json"}
+def add_item(title):
 
+    try:
+        trello_cards_url        = "https://api.trello.com/1/cards"
+        trello_board_list_id_todo = os.environ.get('TRELLO_BOARD_LIST_ID_TODO','')
+        trello_api_key          = os.environ.get('TRELLO_API_KEY','')
+        trello_api_token        = os.environ.get('TRELLO_API_TOKEN','')
+        trello_default_post_header   = {"Accept": "application/json"}
 
-    query = {
-        'idList':   trello_board_list_id_todo,
-        'key':      trello_api_key,
-        'token':    trello_api_token,
-        'name':     title
-    }
-    requests.post(trello_cards_url,headers=trello_default_post_header,params=query,verify=False)
-    get_items()
-    return query
 
+        query = {
+            'idList':   trello_board_list_id_todo,
+            'key':      trello_api_key,
+            'token':    trello_api_token,
+            'name':     title
+        }
+        try:
+            requests.post(trello_cards_url,headers=trello_default_post_header,params=query,verify=False)
+        except:
+            requests.post(trello_cards_url,headers=trello_default_post_header,params=query)
```

Comment:
I'm assuming this was fighting a proxy issue or similar, but do we ever expect this `except` to fire? I can't think why it would fail with `verify=False` but succeed when it does an extra check